test/resources/test_py_version/entry.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import argparse
+import os
+import subprocess
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--output-dir', default=os.environ['SM_OUTPUT_DIR'])
+parser.add_argument('--model_dir')
```

Comment:
you are not using --model_dir so you can just use parser.parse_known_args.

Reply:
actually you don't need to parse the args at all here. Just use SM_OUTPUT_DIR directly.